docs/fomega/lazy-machine/lazy-plutus-core.tex

```diff
@@ -581,7 +581,7 @@ \section{Conclusions}
 when evaluating terms that are ``really'' $n$-ary applications.  Another
 factor is that there are no easily-observable control structures which
 could be handled specially by an evaluator.  Yet another factor is
-than we can't afford to do much in the way of static analysis on-chain,
+then we can't afford to do much in the way of static analysis on-chain,
```

Comment:
hi Ken...yes, you're right!  Since these changes are so minor, do u just want to apply that manually, or should i correct and redo my pull request?  Also, are you interested in stylistic improvements?   Thanks and have a great day!